modules/friendlytag.js

```diff
@@ -168,14 +168,25 @@ Twinkle.tag.callback = function friendlytagCallback() {
 		case 'redirect':
 			Window.setTitle('Redirect tagging');
 
-			form.append({ type: 'header', label: 'Spelling, misspelling, tense and capitalization templates' });
-			form.append({ type: 'checkbox', name: 'redirectTags', list: Twinkle.tag.spellingList });
+			var i = 1;
+			$.each(Twinkle.tag.redirectList, function(title, content) {
+				form.append({ type: 'header', id: 'tagHeader' + i, label: title });
+				var subdiv = form.append({ type: 'div', id: 'tagSubdiv' + i++ });
+				$.each(content, function(subtitle, array) {
```

Comment:
[minor] Can you rename the variable names so that they are more intuitive? `content` -> `group`, `array` -> `subgroup`, `title` -> `groupName`, `subtitle` -> `subgroupName`.

Reply:
Was copied from article, but yeah, much better.  See https://github.com/azatoth/twinkle/pull/1003#discussion_r446517069 as well.